resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+restartfile=<%= EZBake::Config[:restart_file] %>
+timeout="300"
+
+if [ -e $restartfile ];  then
```

Comment:
you'll want to wrap $restartfile in double quotes here to ensure you don't get a bash syntax error if restartfile is not set.